raspberrypi/optimize.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
```

Comment:
this file is intended to be an optional user addition during setup. i would pair this directly beneath curling setup.sh and explain that it adds optional improvements. i can change this to `beeper/beepy/master` if this is something you are interested in actually using.

Reply:
i think there is an open source for the docusaurus for the beeper docs? i can try to add there if i get time if any of this is looking like it will be accepted.

Reply:
i specifically am not running set-cpu-count-to-2.sh. i would reccomend adding that as another option to curl | sudo bash, probably in the getting started page && in the software page that has a few beginnings. the things in that software page should be in a repo and this is a start towards that for this repo.